pallets/gear/src/benchmarking/code.rs

```diff
@@ -784,3 +784,51 @@ impl BitWidth {
         }
     }
 }
+
+// Generate `number` unique WASM function prototypes
+fn generate_uniq_prototypes(number: usize) -> Vec<FuncType> {
+    const ALPHABET: [ValType; 5] = [
+        ValType::I32,
+        ValType::I64,
+        ValType::F32,
+        ValType::F64,
+        ValType::V128,
```

Comment:
f32, f64, v128 are not supported in our contracts

Reply:
You right, this is done to increase number of possible shortest permutation when generating unique types.
So I think it's OK, gonna add a comment.